CMakeLists.txt

```diff
@@ -265,7 +265,7 @@ OCV_OPTION(WITH_KLEIDICV "Use KleidiCV library for ARM platforms" OFF
   VISIBLE_IF (AARCH64 AND (ANDROID OR UNIX AND NOT IOS AND NOT XROS)))
 OCV_OPTION(WITH_NDSRVP "Use Andes RVP extension" (NOT CV_DISABLE_OPTIMIZATION)
   VISIBLE_IF RISCV)
-OCV_OPTION(WITH_HAL_RVV "Use HAL RVV optimizations" (NOT CV_DISABLE_OPTIMIZATION)
+OCV_OPTION(WITH_HAL_RVV "Use HAL RVV optimizations" ON
```

Comment:
Revert this change. Keep `(NOT CV_DISABLE_OPTIMIZATION)`

Reply:
Done